HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -1438,13 +1459,9 @@ void FastTimerService::preSourceEvent(edm::StreamID sid) {
 }
 
 void FastTimerService::postSourceEvent(edm::StreamID sid) {
-  edm::ModuleDescription const& md = callgraph_.source();
-  unsigned int id = md.id();
   auto& stream = streams_[sid];
-  auto& module = stream.modules[id];
 
-  thread().measure_and_store(module.total);
-  ++stream.modules[id].events;
```

Comment:
isn't something like
```c++
++stream.source.events;
```
needed ?

Reply:
I'm currently using `data.events` wherever the number of events for the source is needed so it would be unused anyways. I have no strong preference if you'd rather have `source.events` everywhere instead

Reply:
https://github.com/cms-sw/cmssw/blob/1be9aa532917de9ddf9cc4d69cdf40661afcac02/HLTrigger/Timer/plugins/FastTimerService.cc#L1391-L1392
The "new" source gets the number of events from `data.events` like the other `AtomicResources`